src/elements/OcIcon.vue

```diff
@@ -24,9 +25,15 @@ export default {
      * The name of the icon to display.
      */
     name: {
-      required: true,
+      type: String,
       default: "settings",
```

Comment:
Let's remove the default then? Or put there a condition to use it only if the URL is not specified?

Reply:
taken care of - THX